apps/nextjs-website/src/components/atoms/IconWrapper/IconWrapper.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import * as Icons from '@mui/icons-material';
+import { EIcon } from '@pagopa/pagopa-editorial-components/dist/components/EIcon';
+
+type IconWrapperProps = {
+  iconName: string;
+  color?: string;
+  size?: number;
+};
+
+type IconName = keyof typeof Icons;
+
+const IconWrapper = ({
+  iconName,
+  color = 'primary',
```

Comment:
```suggestion
  color = 'primary.main',
```
i think that the `primary` key, within the theme, corresponds to an object